connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -284,16 +288,18 @@ private void doCommitSync(Map<TopicPartition, OffsetAndMetadata> offsets, int se
      * the write commit.
      **/
     private void doCommit(Map<TopicPartition, OffsetAndMetadata> offsets, boolean closing, final int seqno) {
-        log.info("{} Committing offsets", this);
+        log.info("{} Committing offsets: {}", this, offsets);
         if (closing) {
             doCommitSync(offsets, seqno);
         } else {
             OffsetCommitCallback cb = new OffsetCommitCallback() {
                 @Override
                 public void onComplete(Map<TopicPartition, OffsetAndMetadata> offsets, Exception error) {
+                    log.debug("{} Complete async offset commit callback for generation {}: {}", this, seqno, offsets);
                     onCommitCompleted(error, seqno, offsets);
                 }
             };
+            log.debug("{} Begin async offset commit for generation {}: {}", this, seqno, offsets);
```

Comment:
Actually, I pulled out the logic for the asynchronous commit into a `doCommitAsync` method that is similar to the existing `doCommitSync` method, and changed the `doCommit` to call both of them now. Doing this makes the two cases more similar, and I moved the log message into each of the methods.